mlflow/pyfunc/__init__.py

```diff
@@ -278,29 +278,6 @@ def load_model(model_uri, suppress_warnings=True):
                               loading process will be suppressed. If ``False``, these warning
                               messages will be emitted.
     """
-    return load_pyfunc(model_uri, suppress_warnings)
-
-
-@deprecated("mlflow.pyfunc.load_model", 1.0)
-def load_pyfunc(model_uri, suppress_warnings=False):
```

Comment:
Do we need to keep this around until the next major version (`2.0`) so as not to break any usage dependency?